joern-cli/frontends/c2cpg/src/main/scala/io/joern/c2cpg/astcreation/AstForStatementsCreator.scala

```diff
@@ -294,9 +294,10 @@ trait AstForStatementsCreator { this: AstCreator =>
     val columnNumber = column(statement)
     // We only handle un-parsable macros here for now
     val isFromMacroExpansion = statement.getProblem.getNodeLocations.exists(_.isInstanceOf[IASTMacroExpansionLocation])
+    val code                 = statement.getRawSignature
+    val file                 = Paths.get(statement.getContainingFilename)
     val asts = if (isFromMacroExpansion) {
-      new CdtParser(config, headerFileFinder, None, global)
-        .parse(statement.getRawSignature, Paths.get(statement.getContainingFilename)) match
+      new CdtParser(config, headerFileFinder, None, global).parse(code, file) match
```

Comment:
I'd really prefer having braces over significant whitespace... and that's how we do it normally...

Reply:
Intellij auto-complete bullshit - will fix.